code/modules/pda/PDA.dm

```diff
@@ -300,11 +300,14 @@ GLOBAL_LIST_EMPTY(PDAs)
 		var/obj/item/pen/O = locate() in src
 		if(O)
 			to_chat(user, "<span class='notice'>There is already a pen in \the [src].</span>")
-		else
-			user.drop_item()
-			C.forceMove(src)
-			to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
-			playsound(src, 'sound/machines/pda_button1.ogg', 50, TRUE)
+			return
+		if(C.w_class > WEIGHT_CLASS_TINY)
+			to_chat(user, "<span class='warning'>[C] doesnt fit!</span>")
+			return
+		user.drop_item()
+		C.forceMove(src)
+		to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You slide \the [C] into [src].</span>")
```

Reply:
removing the first /the here also, since it isnt needed

![image](https://user-images.githubusercontent.com/91113370/189514174-952441d8-016d-41d8-9519-2e3216e2ceb2.png)